src/lib/components/NavMenu.svelte

```diff
@@ -107,6 +108,22 @@
 	>
 		Theme
 	</button>
+	{#if $page.data.enableAssistants}
+		<a
+			href="{base}/assistants"
+			class={`flex h-9 flex-none items-center gap-1.5 rounded-lg pl-2.5 pr-2 text-gray-500 hover:bg-gray-100 dark:text-gray-400 dark:hover:bg-gray-700 ${
+				$page.url.pathname === base + "/assistants" && "bg-gray-100 dark:bg-gray-700"
+			}`}
```

Comment:
```suggestion
			class=`flex h-9 flex-none items-center gap-1.5 rounded-lg pl-2.5 pr-2 text-gray-500 hover:bg-gray-100 dark:text-gray-400 dark:hover:bg-gray-700 {
				$page.url.pathname === base + "/assistants" && "bg-gray-100 dark:bg-gray-700"
			}`
```

haven't tested but I think it should work

Reply:
removed it